Workbooks/Virtual Machines/Availability/tmp.tmp

```diff
@@ -0,0 +1 @@
+
```

Comment:
what's this tmp.tmp file?  can you delete this?

Reply:
i needed a file to create the folder, still new at this couldnt find a better way. I did delete it afterwards